web/src/pages/share/[id].vue

```diff
@@ -47,22 +30,18 @@
 
   const loadShareData = async () => {
     const shareId = (route.params as { id: string }).id
+    if (!shareId) return
```

Comment:
Should really redirect back to the planner
```suggestion
    if (!shareId) router.push('/')
```